[Documentation] 英語版READMEを追加 (#47)
approvers/RADWIMPS: README.en.md

# RADWIMPS

<table>
  <tbody>
    <tr>
      <td>
        <a href="README.md">日本語</a>
      </td>
      <td>
        <span>English</span>
      </td>
    </tr>
  </tbody>
</table>

## Rule

> - **Use a method chain** like `then().then().then().世`

Reviewer: ```suggestion
- **Use a method chain** like `then().then().then().世()`
```

> - **Output** `前前前世\n`

Reviewer: CRLF if Windows, LF otherwise?

```suggestion
- **Output** `前前前世` and a line break at the end
```

> - **Use a method chain** like `then().then().then().世()`
> - **Output** `前前前世` and a line break at the end

Reviewer 2: ```suggestion
- **Output** `前前前世` and a line break at end of the line using **method chaining** like this; `then().then().then().世()`
```

Reviewer 2: @raiga0310 
これの確認もよろしくいねがいしま

Reviewer: - `by using` -> `using` じゃだめなん？
- `.` は他の項目についてないから統一したい

Reviewer 2: - `by using` -> `using` 👍
- 全く気づかんかった 変えときます

Reviewer 2: ✔ suggestion 更新済

> - If you have a language feature such as **Promise** or **Future**, **make use of it**

Reviewer 2: ```suggestion
- If you have features of a language such as **Promise** or **Future**, **make use of them**
```
```
Exception in "Rule" part English.grammar.IncorrectArticle: 2
- If you have a language feature such as **Promise** or **Future**, **make use of it**
--------------^-----------------^----------^^^^^^^^^^^^^^^^^^^^^------------------^^--
    at  #Rule(README.en.md:20)
```

Reviewer: これ手動で書いてるとしたらヤバすぎるしかわいい

Reviewer 2: 全手動でございます

> *This repository has nothing to do with the real band RADWIMPS.*

Reviewer 2: ```suggestion
*This repository has nothing to do with the real band "RADWIMPS".*
```
